docs/rst/use_cases/wan_tcp.rst

```diff
@@ -0,0 +1,256 @@
+    *   :ref:`DDS Router <installation_manual_linux>`
+
+2.  Use the `Fast DDS Suite <https://fast-dds.docs.eprosima.com/en/latest/docker/docker_demo/fast_dds_suite.html>`_.
+    This is a publicly available Docker image that contains all the software needed for this tutorial.
+    You can download this Docker image from the `eProsima downloads website <https://www.eprosima.com/index.php/downloads-all>`_.
+
+    To load this image into your Docker repository, from a terminal run
+
+    .. code-block:: bash
+
+        $ docker load -i ubuntu-fastdds-suite:<FastDDS-Version>.tar
+
+    You can run this Docker container as follows
+
+    .. code-block:: bash
+
+        $ docker run -it ubuntu-fastdds-suite:<FastDDS-Version>
+
+Deployment on Net A
+-------------------
+
+First, let's deploy the *Fast DDS Subscriber* and *DDS Router Net A* on a machine on *LAN A*.
+
+Running Fast DDS Subscriber
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1.  Run a Docker container using the Fast DDS Suite image.
+
+    .. code-block:: bash
+
+        docker run -it ubuntu-fastdds-suite:<FastDDS-Version>
+
+1.  Run the Fast DDS Subscriber executing the following commands:
+
+    .. code-block:: bash
+
+        goToExamples
+        ./dds/BasicConfigurationExample/bin/BasicConfigurationExample subscriber --transport udp
+
+
+Running DDS Router Net A
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+As you know, DDS Router is configured through a YAML configuration file.
+Therefore, since we are going to run DDS Router from a Docker container, this file must be shared so that it is accessible within the container.
+
+Let's create first the DDS Router configuration file. It will look like the one shown below.
+
+.. literalinclude:: /resources/use_cases/wan_tcp/dds_router_net_A.yaml
+    :language: yaml
+
+Next, it is briefly explained the most relevant aspects of this configuration file.
+
+The ``participants`` are the interfaces of the DDS Router to communicate with other networks.
+In this case, we have two participants:
+
+    *   ``local``: this is a simple participant that communicates with all DDS entities it finds in the default DDS Domain, domain 0.
+        For more information about this participant please refer to the :ref:`Simple Participant section <user_manual_participants_simple>` of this documentation.
+
+    *   ``wan``: it is a participant designed for WAN communication or the communication between two *DDS Routers*.
+        It uses the `Fast DDS Initial Peers configuration <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/simple.html#initial-peers>`_ to establish a point-to-point communication between two DDS entities, two *DDS Routers* in this case.
+        For now it is not necessary to know more about the details of this configuration as you only need to configure the connection address of the DDS Router.
+
+    For the *DDS Router Net A*, a connection address shall be defined which must be the same as the one exposed by the *DDS Router Net B*.
```

Comment:
If this is supposed to be under list tag `wan`, it is wrongly formatted

Reply:
Wrongly formated.